### PR TITLE
chore: Angular troubleshooting for browser agent

### PR DESCRIPTION
**Pull request by patrickhousley**
Please follow [conventional commit standards](https://www.conventionalcommits.org/en/v1.0.0/)
in your commit messages and pull request title.

## Give us some context

Adding troubleshooting information around Angular 12-14 issue related to the browser agent.

### CODE REVIEW

**Comment by zuluecho9** [comment]
@patrickhousley I added the doc to the browser TW navigation file also. is there anything else this is waiting on that you'll be adding or is it ready to go?

**Comment by patrickhousley** [comment]
Thanks @zuluecho9 I just want to get 👀 from my team first.

File: src/content/docs/browser/single-page-app-monitoring/troubleshooting/angular-truncated-copy-paste-snippet.mdx [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+## Problem
+
+You are using the [HTML copy/paste method](/docs/browser/new-relic-browser/installation-configuration/adding-apps-new-relic-browser/#copy-paste-app) within an Angular version 12, 13, or 14 application, but the agent code snippet has been truncated after running the Angular build.
+
+This may cause the affected application or pages to no longer function correctly due to improper JavaScript contained within the HTML of the page.
+
+## Solution
+
+This issue is caused by a bug within a dependency the Angular CLI build system (versions 12, 13, and 14) rely upon in combination with the browser agent version 1225 and higher. The build has a limitation in the size of a script contained within a script element. We have confirmed this has been corrected with Angular version 15 and recommend upgrading your Angular project.
```
Comment (metal-messiah) [inline src/content/docs/browser/single-page-app-monitoring/troubleshooting/angular-truncated-copy-paste-snippet.mdx:18]: (IMO) This first paragraph here kind of flows awkwardly and is a little too specific to scale over time.

```suggestion
This issue is caused by a bug within a dependency the Angular CLI build system. Versions 12, 13, and 14 of the Angular CLI use a dependency that truncates inline scripts of an arbitrary size, which can include the New Relic Browser agent. We have confirmed this has been corrected with Angular version 15 and recommend upgrading your Angular project.
```

**Comment by patrickhousley** [comment]
@zuluecho9 this is ready to ship.

File: src/content/docs/browser/single-page-app-monitoring/get-started/spa-compatibility-requirements.mdx [hunk 2]
```diff
@@ -30,6 +30,7 @@ Below are known compatibility issues:
 
 * If your application uses AngularJS and you want to use browser's [SPA monitoring](/docs/browser/single-page-app-monitoring/get-started/welcome-single-page-app-monitoring) capabilities, Zone.js versions 0.6.18-0.6.24 are not compatible with the SPA agent.
 * The html2pdf.js library is not compatible with the SPA agent.
+* Angular versions 12 through 14 contains a limitation around the size of individual script elements contained within the index HTML source. See our [troubleshooting](/docs/browser/single-page-app-monitoring/troubleshooting/angular-truncated-copy-paste-snippet) for more information.
```
Comment (ally-sassman) [inline src/content/docs/browser/single-page-app-monitoring/get-started/spa-compatibility-requirements.mdx:33]:
```suggestion
* Angular versions 12 through 14 contains a limitation around the size of individual script elements contained within the index HTML source. See [this troubleshooting page](/docs/browser/single-page-app-monitoring/troubleshooting/angular-truncated-copy-paste-snippet) for more information.
```

**Pull request merged**